scripts/battery3

```diff
@@ -51,7 +51,9 @@ if [ -z "$CHARGE_STATE" ]; then
     elif [ "$CHARGE_STATE" == "discharging" ]; then
     CHARGE_ICON=$(xrescat i3xrocks.label.dn D)
     elif [ "$CHARGE_STATE" == "charging" ]; then
-    CHARGE_ICON=$(xrescat i3xrocks.label.up C)
+    CHARGE_ICON=$(xrescat i3xrocks.label.batterycharging C)
```

Comment:
These Xresource keys have underscores.  See https://github.com/regolith-linux/regolith-styles/blob/master/cahuella/typeface#L18-L19

Reply:
Ah, yeah the labels I have was from 1.2 release that's why there were no underscore. I'll update the PR. Thanks!